scripts/setup-helper-functions.sh

```diff
@@ -100,7 +100,7 @@ function get_cxx_flags {
     ;;
 
     "avx")
-      echo -n "-mavx2 -mfma -mavx -mf16c -masm=intel -mlzcnt -std=c++17"
```

Comment:
Can we actually remove the flag ? Does it make any material difference ?

Reply:
> Can we actually remove the flag ? Does it make any material difference ?
masm is used to specify the syntax of assembly. Actually I have no idea why it was added. But setting it to intel will break folly build and thrift library used by presto_cpp. Those two project require masm to be att. As the masm defaults to 'att' so it's ok for me to remove the flag here and probably let each component to determine to use a different setting of masm by calling out masm in its own build.